hw4/01-routing.js

```diff
@@ -1,49 +1,107 @@
-const express = require('express');
-const app = express();
+const http = require("http");
 const port = process.env.PORT || 5000;
```

Comment:
**01-routing.js**
Looks fantastic to me! Everything works correctly and I get all the correct responses. The only tiny thing I might suggest is to use ` 5001` instead of `5000` for the port. The professor mentioned that the `ControlCe` service on OSX usually uses 5000, so sometimes OSX users can get a "Already in use" error. Everything else looks great!